dev_guide/application_lifecycle.adoc

```diff
@@ -0,0 +1,77 @@
+= The Application Life Cycle
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+As a PaaS, OpenShift is built for deploying applications. Depending on how much
+you want to involve OpenShift in the development process, you can choose to
+develop on OpenShift after it has been installed, and use it to continuously
+develop an application, or you can deploy a fully developed application onto a
+newly installed OpenShift instance.
+
+== Developing on OpenShift
+
+If you plan to use develop your application on OpenShift directly:
+
+Plan::
+* What does your application do?
+* What programming language will you be developing in?
+ 
+Install::
+* OpenShift should be installed by this point, either by yourself or an
+administrator within your organization.
+
+Develop::
+* This is where you develop your application code.
+
+Generate::
+* link:../dev_guide/new_app.html[Create an basic application] so that OpenShift can determine the programming language you are using. OpenShift generates build and deployment configurations.
+
+Code/Manage/Verify::
+* How will you continue to develop your application? 
+* Remember to periodically verify that your application code is running
+successfully.
```

Comment:
this step needs a little more detail i think.  like how does one "verify"  (the answer is:  you push your changes to your git repo and tell openshift to rebuild/redeploy your application.  or you use rsync to sync code changes into your running application pod, if your application+framework support hot deploy).

it's also possible you do some verification locally (eg I'm coding a java app in eclipse, i periodically run some unit tests locally, and then when i'm really confident, i push my changes to git and build/deploy the whole updated application on openshift and test that)